### PR TITLE
Update release channel language / tables

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/359

Attempting to both:

- Clarify which releases are stable and which are LTS
- Clarify that LTS releases can have new features and stable releases can have testing (without over-explaining and making the channels seem arbitrary)

If these changes look good, then I'll push them to other Enterprise / Cloud versions with similar language

### CODE REVIEW

File: enterprise/ac-support-policy.md [hunk 1]
```diff
@@ -114,15 +114,20 @@ Maintenance is discontinued the last day of the month for a given version. For e
...
 | [2.0](https://github.com/astronomer/ap-airflow/blob/master/2.0.0/CHANGELOG.md)     | Dec 17, 2020   | February 2022*         |
-| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md) (LTS)     | May 21, 2021   | November 2022           |
 | [2.2](https://github.com/astronomer/ap-airflow/blob/master/2.2.0/CHANGELOG.md)     | Oct 12, 2021   | April 2022              |
 
 > *Given the wide usage of AC 1.10.12, 1.10.14, and 2.0, Astronomer has extended the maintenance period for those versions through February 2022.
 
+### LTS Releases
+
+| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) (LTS) | March 19, 2021 | September 2022         |
+| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md) (LTS)     | May 21, 2021   | November 2022           |
```
Comment (jwitz) [inline enterprise/ac-support-policy.md:131]:
```suggestion
| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022         |
| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)     | May 21, 2021   | November 2022           |
```

**Comment by jwitz** [comment]
Took @paolaperaza 's suggestion over Zoom to consolidate the tables again and instead add a "Release channel" column. This is because we expect that customers are more likely to look for a specific Airflow version than to check the releases in a specific release channel.

**Review by paolaperaza — COMMENTED** [review]
Hey @jwitz - I think I narrowed in on some updated language here on how we want to talk about release channels. The main difference is that a release is not either OR - it's just stable _plus_ LTS.

Let's not spend more time word smithing for now - if you could just pass on the AC specific language (not Runtime) to the rest of the Software versioned docs, that'd be great.

**Review by paolaperaza — APPROVED** [review]
Actually, JK - just went ahead and did it myself. Going to merge! If we get more feedback on Stable vs. LTS confusion we can revisit.

**Pull request merged**